### PR TITLE
write_stylus: add desktop icon

### PR DESCRIPTION
**Pull request by oyren**
Write has no icons on its own, that's why I have added a freely available icon. Also I fixed a typo.
###### Motivation for this change
I didn't like that write didn't have an icon in the application launcher.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/graphics/write_stylus/default.nix [hunk 1]
```diff
@@ -16,6 +17,12 @@ stdenv.mkDerivation rec {
     url = "http://www.styluslabs.com/write/write${version}.tar.gz";
     sha256 = "1p6glp4vdpwl8hmhypayc4cvs3j9jfmjfhhrgqm2xkgl5bfbv2qd";
   };
+
+  icon = fetchurl {
+    url = "https://oyra.eu/write/icon.tar.gz";
```
Comment (Mic92) [inline pkgs/applications/graphics/write_stylus/default.nix]: What is the license of this file. Is it your own creation or what is the origin?

Comment (Mic92) [reply pkgs/applications/graphics/write_stylus/default.nix]: Remember not to change/remove this file.

Comment (oyren) [reply pkgs/applications/graphics/write_stylus/default.nix]: Sure.

Comment (voidIess) [reply pkgs/applications/graphics/write_stylus/default.nix]: > Remember not to change/remove this file.

tl;dr
the file was removed and the package is uninstallable now...

Comment (oyren) [reply pkgs/applications/graphics/write_stylus/default.nix]: True, it's on my todo list but i haven't found the time yet.

Comment (oyren) [reply pkgs/applications/graphics/write_stylus/default.nix]: #60090
